events/hfe_events_primitive.txt

```diff
@@ -75,3 +76,495 @@ country_event = {
 		}
 	}
 }
+
+# Keeps the High Mechanicus' Civics on point
+country_event = { # Default, their civilization continues naturally
+	id = hfe_primitive.1
+	hide_window = yes
+
+	trigger = {
+		has_country_flag = hfe_varelse_primitive_0
+		OR = {
+			NOT = { is_country_type = primitive }
+			has_country_flag = primitives_can_into_space
+			has_country_flag = high_mechanicus_can_into_space
+		}
+		OR = {
+			has_country_flag = high_mechanicus_retained
+			has_civic = hfe_civic_high_mechanicus
+			is_spiritualist = yes
+		}
+		NOT = {
+			has_country_flag = hfe_varelse_fedora_governance
+			has_civic = hfe_civic_overthrown_mechanicus
+			is_materialist = yes
+		}
+		NOT = { has_country_flag = high_mechanicus_befriended }
+		NOT = { has_country_flag = high_mechanicus_can_into_space }
+	}
+
+	mean_time_to_happen = {
+		days = 4
```

Comment:
Lol no

Reply:
I mean no mtth in this mod

Reply:
Okay, there's a reason behind this. There is no On-Action for Primitives becoming FTL Empires.

When a Primitive Planet achieves FTL flight, it deletes its Civics and its Authority and randomly creates new ones among the Vanilla rendition without any rhyme or reason. That includes the High Mechanicus. 

What this Event does is fix that. Four days after Colony Beta achieves spaceflight, this Event fires and changes the High Mechanicus back to what it was meant to be.

If you can teach me some sort of way to make this happen WITHOUT Mean Time to Happen, I will immediately make the change. Until then, it stays.

Reply:
Had, if there is a way to do this without Mean Time to Happen, tell me and I will fix it.

Reply:
Had, if you don't have a solution to this problem, then the MTTH will have to stay. If there is a way to do this without any MTTH triggers, please tell me.